packages/graphql/index.js

```diff
@@ -1,42 +1,12 @@
 // @flow
 
-import {
-  graphql,
-  GraphQLSchema,
-  GraphQLObjectType,
-  GraphQLString,
-} from 'graphql';
-import GlobalID from '@kiwicom/graphql-global-id';
+import '@babel/polyfill';
 
-import fetch from './src/services/Fetch';
-import Itineraries from './src/queries/Itineraries';
-import createContext from './src/services/GraphQLContext';
-
-const schema = new GraphQLSchema({
-  query: new GraphQLObjectType({
-    name: 'RootQueryType',
-    fields: {
-      searchItineraries: Itineraries,
-      id: GlobalID(() => 'TODO: implement me'),
-
-      hello: {
-        type: GraphQLString,
+import { ApolloServer } from 'apollo-server';
 
-        async resolve() {
-          const test = await fetch('/locations/query?term=oslo');
-
-          return `Welcome to Tequila client demo! Test location query = ${
-            test.locations[0].id
-          }`;
-        },
-      },
-    },
-  }),
-});
+import createContext from './src/services/GraphQLContext';
+import schema from './src/Schema';
 
-const inMemoryFetcher = (source: string, variableValues: {||}) => {
-  const context = createContext();
-  return graphql(schema, source, undefined, context, variableValues);
-};
+const server = new ApolloServer({ schema, context: createContext() });
 
-export { schema, inMemoryFetcher };
+export { schema, server };
```

Comment:
I am not so sure we should export server from this file. No other app will use it. I think you should move the creation of the server to the server.js file. 

And this file only exports what other packages may use.

Reply:
make sense, I'll do how you suggested thanks.

Reply:
done